galley/pkg/server/server.go

```diff
@@ -171,6 +173,7 @@ func newServer(a *Args, p patchTable) (*Server, error) {
 		Watcher:            distributor,
 		Reporter:           s.reporter,
 		CollectionsOptions: source.CollectionOptionsFromSlice(metadata.Types.Collections()),
+		ConnRateLimiter:    mcprate.NewRateLimiter(time.Second, 100), // TODO(Nino-K): These should be configurable
```

Comment:
Can you reference a GH issue in the TODO for better tracking? We don't directly track TODOs in the code.